app/javascript/gobierto_data/webapp/components/commons/Visualizations.vue

```diff
@@ -92,10 +92,16 @@ export default {
       /* We compare the columns, if it returns true we pass the schema to Perspective, if false Perspective will convert the columns*/
       const sameColumns = arrayColumnsFromAPI.length === arrayColumnsFromQuery.length && arrayColumnsFromAPI.every(column => arrayColumnsFromQuery.includes(column))
 
+      //Check if any columns contains Boolean values
+      const datasetsHasBooleanColumns = this.checkIfBooleanExist(this.objectColumns)
+
+      //If columns contains Boolean values goes to replace them
+      let replaceItems = datasetsHasBooleanColumns.length > 0 ? this.replaceBooleanValues(this.items) : this.items
```

Comment:
You complicate things a lot

```suggestion
      let replaceItems = this.items
      if (this.objectColumns.some(({ value }) => value === "boolean" )) {
        replaceItems = this.items.replace(/"t"/g, '"true"').replace(/"f"/g, '"false"')
      }
```

Reply:
And remove the `checkIfBooleanExist` function, sure